app/controllers/subscriptions_controller.rb

```diff
@@ -0,0 +1,24 @@
+require_relative "../models/mailchimp"
+
+class SubscriptionsController < ApplicationController
+  # Subscribe to the MailChimp newsletter.
+  def create
+    redirect_to request.referer if !current_user
```

Comment:
@kytrinyx Done in https://github.com/ossfriday/ossfriday/pull/68 so `script/cibuild` will always run both tests and rubocop even if the first fails.